packages/minter-contracts/ligo/src/minter_collection/editions/README.md

```diff
@@ -55,17 +57,17 @@ editions_metadata :=
 - `distribute_editions : list (edition_id, receivers : nat * (address list))`
     For each `Edition run` corresponding to a given `edition_id`, `editions` are distributed to the addresses in `receivers`. Each distribution mints a new token to some `token_id` in the `token_id`s reserved for that edition when `mint_edition` was called.  
 
-  + Only a creator of some edition run can distribute or mint to `token_id`s belonging to their `edition run` and the contract must NOT be paused.
+  + Only a creator of some edition run can distribute or mint to `token_id`s belonging to their `edition run`.
```

Comment:
Is the creator the same as the admin?

Reply:
Yes unless admin is changed

Reply:
And what happens then? Does the new admin "inherits" all the permissions from the old one?

Reply:
My expectation is that the new admin should only gain the ability to mint / mint editions. They should not gain the ability to distribute editions for the old admin. I will clarify this in the spec if you agree.

Reply:
Does it mean that if there are undistributed editing and the admin changes, nobody can distribute any more.
I think it would more clear if each entry point would specify what are the guards there.

Reply:
No the creator can still distribute even if they are not admin anymore. I agree, I will add that.